src/sass/components/_articles.scss

```
@@ -0,0 +1,34 @@
+.articles {
+  padding: 56px 0 40px 0;
+
+  &__title {
+    text-align: center;
+  }
+
+  &__container {
+    position: relative;
+  }
+
+  &__cards {
+    @include flex(column, flex-start, flex-start, nowrap);
+    max-height: 1950px;
+    overflow: hidden;
+  }
+}
+
+.card {
+  @include flex(column, center, center, nowrap);
+
+  &__title {
+    margin: 20px 0 42px 0;
```

and a shorthand here. it's not bad that you don't use it, I just want to make sure that you know that you CAN use it
```suggestion
    margin: 20px 0 42px;
```